src/ruisapp/glue/windows/glue.cxx

```diff
@@ -821,8 +821,14 @@ int WINAPI WinMain(
 	int n_cmd_show // Window Show State
 )
 {
-	// TODO: pass argc and argv
-	ruisapp::winmain(0, nullptr);
+	ruisapp::winmain(__argc, (const char**)__argv);
+
+	return 0;
+}
+
+int main(int argc, char** argv)
```

Comment:
I think here you can mark the argv argument as `const char**` and then no need to cast